web-ts/src/components/GeneralHeader.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import landingIcon from '../assets/location-icon.svg'
+import Header from './Header'
+import HeaderActionItemLink from './HeaderActionItemLink'
+import { withTranslation, TFunction } from 'react-i18next'
+import buildConfig from '../constants/buildConfig'
+
+type PropsType = {
+  viewportSmall: boolean
+  t: TFunction
+}
+
+const GeneralHeader = ({ viewportSmall, t }: PropsType) => {
+  // const getPath = new I18nRedirectRouteConfig().getRoutePath
+  // TODO use right path
```

Comment:
I'm,  a little bit afaid we will miss some of the todos if there are no issue link. We could also use TODO-REFACTORING or FIXME from now on to keep track.

Reply:
We got it :) We can just check IntelliJ TODOs tab and search afterwards. I have it on my TODO list as well :grin: 
This is also already fixed in #305.

Reply:
Created the following issue for other todos regarding checking whether url matches a route: https://issues.tuerantuer.org/browse/IGAPP-668